test-workspace/vault/dendron.ref.assets.md

```diff
@@ -0,0 +1,14 @@
+---
+id: Hf27I1UR3HvKyd6HRh8C0
+title: Assets
+desc: ''
+updated: 1637576810331
+created: 1637568178090
+---
+
+## PDF pasted with `Dendron: Paste File` command
+[dummy-pdf.pdf](assets/dummy-pdf.pdf) 
```

Comment:
should also add case of `./assets/` and `/assets/`.

also, can we add a vault or rename one of the local vaults to `assets` and create a cross vault link to it?

Reply:
Added 

New vault `assets` and asset link to it from `vault` file 
```
* [test-file-in-another-vault-which-is-named-assets](dendron://assets/assets/assets/test-file-in-asset-vault.md)
```